src/itest/kotlin/nl/lifely/zac/itest/AanvullendeInformatieTaskListenerTest.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.lifely.zac.itest
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.kotest.assertions.json.shouldBeJsonObject
+import io.kotest.assertions.json.shouldContainJsonKeyValue
+import io.kotest.core.spec.Order
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import nl.lifely.zac.itest.client.ItestHttpClient
+import nl.lifely.zac.itest.client.ZacClient
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_SPEC_ORDER_AFTER_SEARCH
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.zaakProductaanvraag1Uuid
+import org.json.JSONArray
+
+/**
+ * This test assumes a human task plan item (=task) has been started for a zaak in a previously run test.
+ */
+@Order(TEST_SPEC_ORDER_AFTER_SEARCH)
+class AanvullendeInformatieTaskListenerTest : BehaviorSpec({
+    val logger = KotlinLogging.logger {}
+    val itestHttpClient = ItestHttpClient()
+    val zacClient = ZacClient()
+
+    Given("A zaak with one Aanvullende informatie task") {
```

Comment:
Have you also tested/does this also work when you have multiple 'aanvullende informatie' tasks and you close first one, and then also the other one?

Reply:
ah cool. nice!